public/scripts/world-info.js

```diff
@@ -3189,6 +3192,17 @@ function handleEntryKillSwitchHelper({ entryKillSwitch, entry, data, name, templ
     entryKillSwitch.toggleClass('fa-toggle-on', isActive);
 }
 
+/**
+ * Update commentInput's placeholder.
+ * @param {string} keys Text to display in commentInput's placeholder.
+ * @param {JQuery<HTMLElement>} commentInput
+ */
+function setCommentPlaceholder(keys, commentInput) {
+    //Show keys in the placeholder.
+    keys = keys.slice(0,1000);
+    commentInput.attr('placeholder', (keys !== '' ? keys : 'Entry Title/Memo'));
```

Comment:
The added text is not translatable.

Also it will be easier to do it like

```suggestion
    commentInput.attr('placeholder', (keys || t`Entry Title/Memo`));
```

Reply:
Thank you.
https://github.com/SillyTavern/SillyTavern/pull/4681/commits/e9528faef8ea356b41619c16baae0f34577bec07